src/main/java/seedu/duke/util/Parser.java

```diff
@@ -425,6 +427,7 @@ private Command prepareDeleteNote(String userMessage) throws SystemException {
                 }
             }
 
+            // If prefix is title, delete note by title. Else delete index.
```

Comment:
Actually you can return a new DeleteNoteCommand within the switch case. If it reaches the code after the switch case can throw a systemException, say missing title/index or missing description

Reply:
Good idea. Shall implement it right away. Thank you for your suggestion.